### ISSUE TITLE
Autoplay specific button only (PIU center button for example)

### ISSUE THREAD

**Issue by skyli91** [stepmania/stepmania#1156]
This could be useful for anyone wanting to play PIU on a DDR dance pad (autoplay on center button) or when one of the buttons/panels is broken, it could be autoplayed to be able to dance. Pump it up would benefit from this the most with autoplay on the center button.

**Comment by kyzentun** [comment]
This can also be useful for auto-scratch mode in beatmania style game types.

**Comment by 1033Forest** [comment]
It could also be useful, if one of the panels on a pad is invalid and/or broken.

**Comment by kyzentun** [comment]
I will add that if you're playing on pad, turning off one panel will not result in a chart that is fun and makes sense.

**Comment by 1033Forest** [comment]
Yeah... Now I think I know the reason why not many people play or care about the 3 panel mode, and why it doesn't have a button on the select style menu yet.

I only play 3 panel for fun or if I'm really bored. I usually stick with the normal 4 panel mode.
Para mode, is just solo without the down arrow. KB7 and Beat are similar, it's just KB7 has a space button instead of a scratch thingy. Some modes are just other modes with one arrow added or missing.

**Comment by SoonDead** [comment]
Great idea! I would not recommend using it to skip steps, but to mitigate a missing or broken button.

**Comment by mikeyrorymac** [comment]
My down arrow is broken and this would be a really useful compromise until I get it repaired. Dropping combos every 4th step is not fun.
